backend/src/openbeheer/utils/gherkin_e2e.py

```diff
@@ -176,6 +176,9 @@ def user_logs_out(self, page: Page) -> None:
 
             logout_button = page.get_by_role("button", name="Logout")
             logout_button.wait_for()
+            page.wait_for_timeout(
+                120
+            )  # ¯\_(ツ)_/¯ - Attempt to fix flakiness in WebKit
```

Comment:
Since it's [discouraged](https://playwright.dev/python/docs/api/class-frame#frame-wait-for-timeout) I'd make it conditional on browser_name
```suggestion
            if self.browser_name == "webkit":
                # ¯\_(ツ)_/¯ - Attempt to fix flakiness in WebKit
                page.wait_for_timeout(120)  
```

Just need to add `browser_name` fixture. Maybe you can add it to `conftest.runner` and pass it to the scenario constructor.

That way we can mark the webkit hacks as such.